app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -1,4 +1,4 @@
-$LOAD_PATH << File.join(GEMS_PENDING_ROOT, "Scvmm")
+# encoding: US-ASCII
```

Comment:
Because I was under the mistaken impression that was standard practice in our code, but now I see that not to be the case.